api/src/feature/onboarding/resolver/verifyPhoneNumberCode.ts

```diff
@@ -1,15 +1,19 @@
 import { MutationResolvers } from "@corecodeio/libraries/api";
 import { IContext } from "../../../server/interface/IContext";
 import { OnboardingControllerInjectionKey } from "../InjectionKeys";
+import { VerfyPhonNumber } from '../../../middleware/VerifyPhoneNumber';
 
 export const verifyPhoneNumberCode: MutationResolvers<
   IContext
 >["verifyPhoneNumberCode"] = (parent, { input }, { dependencies }) => {
   try {
-    const onboardingController = dependencies.provide(
-      OnboardingControllerInjectionKey
-    );
-
-    return onboardingController.verifyPhoneNumberCode(input);
+    const verfyPhonNumber = VerfyPhonNumber(input)
+    if (verfyPhonNumber.length > 0 ){
```

Comment:
Estamos usando `prettier`?

También, más cuidado con la nomenclatura:

```suggestion
    if (verifyPhoneNumber.length > 0) {
```

Reply:
ya esta arreglado